docs/source/develop.rst

```diff
@@ -52,8 +52,8 @@ non-exhaustive list follows:
 
 Git and GitHub can be challenging at first.  Fortunately good materials exist
 on the internet.  Rather than repeat these materials here, we refer you to
-Pandas' documentation and links on this subject at
-https://pandas.pydata.org/pandas-docs/stable/contributing.html
+pandas' documentation and links on this subject at
+https://pandas.pydata.org/pandas-docs/stable/development/contributing.html
```

Comment:
We could also point to the dev-docs. We adjusted things quite regularly recently.

Reply:
I'll defer to you as to which link would be a better source for contributors

Reply:
The lets point to the dev version, bugs are fixed sooner there